sites/public/src/components/applications/HouseholdMemberForm.tsx

```diff
@@ -34,12 +35,9 @@ const HouseholdMemberForm = (props: HouseholdMemberFormProps) => {
             {props.strings?.edit ?? t("t.edit")}
           </Button>
         ) : (
-          <Icon
-            className="ml-2 pt-4 block"
-            size="medium"
-            symbol="lock"
-            fill={IconFillColors.primary}
-          />
+          <Icon className="ml-2 mt-4 text-primary" size="md">
```

Comment:
I think this is okay for now, but we're trying to not add any new tailwind styles as we move away from tailwind, and are adding module files as necessary. This won't be the seeds primary color. I know we chatted about it briefly but we might want some utility classes in seeds.